cmd/config/config.go

```diff
@@ -54,7 +55,31 @@ func run() func(*cobra.Command, []string) error {
 
 			fmt.Fprint(cmd.OutOrStdout(), string(configJSON)+"\n")
 		case viper.GetBool(SetFlag):
-			fmt.Fprintln(cmd.OutOrStdout(), "called --set flag")
+			// flag needs two arguments: a key and value
+			if len(args)%2 != 0 {
+				return errors.New("flag needs an argument: --set")
```

Comment:
This is the same error that cobra shows for a flag that requires a single value.